examples/proxy/record-encryption/cluster-ip/proxy/proxy-config.yaml

```diff
@@ -29,11 +29,21 @@ data:
     filters:
     - type: RecordEncryption
       config:
-        kms: VaultKmsService
+        # User: uncomment this section if you are using AWS.
+        kms: AwsKmsService
```

Comment:
Unfortunately Kustomize can't handle patches within configmap content.  If this feature ever lands, this will get nicer https://github.com/kubernetes-sigs/kustomize/issues/5109

Reply:
I think your right about it being tolerable, but pedantically I think it should start with both blocks commented out.